frontend_readme.md

```diff
@@ -0,0 +1,57 @@
+# **Productivity Cube - frontend**
+
+## Description
+This is a frontend part of Time Management Supporting System. The whole idea is about helping user with making decisions on what he should put his focus at the moment. We create a small cube, which analyzes our day at work and then
+predict if the action taken during some part of the day would be efficient or will be interrupted
+by other factors.
+
+
+## Installing
+bash
+```
+install dependencies

```

Comment:
before that this service requires to `cp .env.example .env` , defining VUE_APP_API_URL variable and starting backend project (please add a link to its readme file'